src/TConsole.cpp

```diff
@@ -1096,6 +1096,11 @@ void TConsole::scrollUp(int lines)
 
     if (lowerAppears) {
         QTimer::singleShot(0, this, [this]() {  mUpperPane->scrollUp(mLowerPane->getRowCount()); });
+        if (!mpHost->mTutorialForSplitscreenScrollbackAlreadyShown) {
+            QString infoMsg = tr("[ INFO ] - Split-screen scrollback activated. Press CTRL-ENTER to cancel.");
```

Comment:
We need to improve the design of this function... but for now one more space is needed

```suggestion
            QString infoMsg = tr("[ INFO ]  - Split-screen scrollback activated. Press CTRL-ENTER to cancel.");
```